src/wallet/wallet.cpp

```diff
@@ -2381,6 +2381,8 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const CoinEligibil
         temp.m_confirm_target = 1008;
         CFeeRate long_term_feerate = GetMinimumFeeRate(*this, temp, &feeCalc);
 
+        std::vector<OutputGroup> groups = GroupOutputs(coins, !coin_selection_params.m_avoid_partial_spends, eligibility_filter.max_ancestors);
```

Comment:
in 6148a8acda5e594bb9b3b2d989056f9e03ddbdbd

This line seems to be the same in both if-else blocks so it could be moved to the beginning of the function before the if

Reply:
Ah, never mind, it changes later when `positive_only` is added